syntax = "proto3";

package dml;

import "node_def.proto";

service Worker {
  rpc Init (InitRequest) returns (InitResponse) {}
}

message InitRequest {
  repeated NodeDef def = 1;
  int32 iterations = 2;
}

message InitResponse {
  string message = 1;
}